x/liquidstakeibc/keeper/ica.go

```diff
@@ -73,7 +71,7 @@ func (k *Keeper) GenerateAndExecuteICATx(
 	}
 	k.Logger(ctx).Info(
 		fmt.Sprintf(
-			"sent ICA transactions with seq: %v, connectionID: %s, ownerID: %s, msgs: %s",
+			"Sent ICA transactions with seq: %v, connectionID: %s, ownerID: %s, msgs: %s",
 			msgSendTxResponse.Sequence,
```

Comment:
i guess logs like these can also be converted to keyvals? reduces usage of sprintf.

Reply:
missed this one